internal/xds/bootstrap/bootstrap.yaml.tpl

```diff
@@ -166,6 +171,23 @@ static_resources:
                 address: {{ $sink.Address }}
                 port_value: {{ $sink.Port }}
   {{- end }}
+  - connect_timeout: 10s
```

Comment:
is it safe to add this by default without a `{{- if .ZoneDiscoveryEnabled  }}` container  ? @nezdolik @mathetake

Reply:
Yes the new local_cluster would just render as having zero endpoints and wouldn't change any existing behavior.

Reply:
Yes the new local_cluster would just render as having zero endpoints and wouldn't change any existing behavior.